dist/backdraft.js

```diff
@@ -1275,8 +1275,18 @@ _.extend(Plugin.factory, {
 
     render : function() {
       var cells = this.findCells(), node;
-      _.each(this.columnsConfig, function(config) {
-        node = cells.filter(config.nodeMatcher(config));
+      var cleanConfigs = this.columnsConfig.map(function(currentValue,index){
```

Comment:
How about visibleColumnConfigs? I think that matches up with what you are doing here, trying to get the indexes of the visible columns.

Also, seems like this could be extracted into a function to make the code easier to read.